src/Microsoft.IdentityModel.Validators/AadIssuerValidator/AadIssuerValidator.cs

```diff
@@ -125,7 +125,7 @@ public class AadIssuerValidator
                     {
                         if (IsV2Authority)
                         {
-                            AadIssuerV2 = validationParameters.Configuration.Issuer;
+                            AadIssuerV2 = validationParameters.Configuration != null ? validationParameters.Configuration.Issuer : CreateConfigManager(AadAuthorityV2).GetConfigurationAsync().ConfigureAwait(false).GetAwaiter().GetResult().Issuer;
```

Comment:
FYI: This PR removes Configuration from the validation parameters: https://github.com/AzureAD/azure-activedirectory-identitymodel-extensions-for-dotnet/pull/1779